src/clients/index.test.js

```diff
@@ -205,80 +317,88 @@ describe('SplitsterClient', () => {
+          banner: 'winningVariant',
+          loginButton: 'winningVariant',
+          homepage: '__disabled_because it is not ready yet'
+        });
       });
-    });
-    test('or fallback disabled reason (if not provided) for each test', () => {
-      mockSeedRandom.mockImplementation(BASIC_CONFIG_WITH_DISABLED_WITH_DEFAULT_VARIANT.mockSeedRandomImplementation)
-      const client = initClient(BASIC_CONFIG_WITH_DISABLED_WITH_DEFAULT_VARIANT.config);
-      expect(client.getSaveResults()).toEqual({
-        banner: 'winningVariant',
-        loginButton: 'winningVariant',
-        homepage: '__disabled_config'
+      test('or fallback disabled reason (if not provided) for each test', () => {
+        mockSeedRandom.mockImplementation(
+          BASIC_CONFIG_WITH_DISABLED_WITH_DEFAULT_VARIANT.mockSeedRandomImplementation
+        );
+        const client = initClient(
+          BASIC_CONFIG_WITH_DISABLED_WITH_DEFAULT_VARIANT.config
+        );
+        expect(client.getSaveResults()).toEqual({
+          banner: 'winningVariant',
+          loginButton: 'winningVariant',
+          homepage: '__disabled_config'
+        });
       });
-    });
-    test('with the version in the testId if requested', () => {
-      mockSeedRandom.mockImplementation(BASIC_CONFIG_WITH_VERSION.mockSeedRandomImplementation)
-      const client = initClient(BASIC_CONFIG_WITH_VERSION.config);
-      expect(client.getSaveResults(true)).toEqual({
-        banner_0: 'winningVariant',
-        loginButton_5: 'superdupercool',
-        homepage_0: 'winningVariant'
+      test('with the version in the testId if requested', () => {
+        mockSeedRandom.mockImplementation(
+          BASIC_CONFIG_WITH_VERSION.mockSeedRandomImplementation
+        );
+        const client = initClient(BASIC_CONFIG_WITH_VERSION.config);
+        expect(client.getSaveResults(true)).toEqual({
+          banner_0: 'winningVariant',
+          loginButton_5: 'superdupercool',
+          homepage_0: 'winningVariant'
+        });
       });
-    });
-    test('with the default variant if the test is disabled, and noDisabled option is set to true', () => {
-      mockSeedRandom.mockImplementation(BASIC_CONFIG_WITH_DISABLED_WITH_DEFAULT_VARIANT.mockSeedRandomImplementation)
-      const client = initClient(BASIC_CONFIG_WITH_DISABLED_WITH_DEFAULT_VARIANT.config);
-      expect(client.getSaveResults(false, true)).toEqual({
-        banner: 'winningVariant',
-        loginButton: 'winningVariant',
-        homepage: 'default'
+      test('with the default variant if the test is disabled, and noDisabled option is set to true', () => {
+        mockSeedRandom.mockImplementation(
+          BASIC_CONFIG_WITH_DISABLED_WITH_DEFAULT_VARIANT.mockSeedRandomImplementation
+        );
+        const client = initClient(
+          BASIC_CONFIG_WITH_DISABLED_WITH_DEFAULT_VARIANT.config
+        );
+        expect(client.getSaveResults(false, true)).toEqual({
```

Comment:
I would consider using const for function parameters so it's even more visible from test code only what first and second boolean params mean

Reply:
I would prefer a more radical approach to be honest :)

I would like to modify the signature of these functions to accept only one argument, being an object, so that all properties are named when calling them.

This would be done in a different iteration. WDYT?

Reply:
👍